server/controllers/partiesController.js

```diff
@@ -43,6 +43,31 @@ class partyController {
       error: 'Party not found',
     });
   }
+
+  static addNew(req, res) {
```

Comment:
Add new what? Give a better variable naming

Reply:
Copy that.